libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -129,37 +176,51 @@ static int fill_model_input_ov(OVModel *ov_model, OVRequestItem *request)
     lltask = ff_queue_peek_front(ov_model->lltask_queue);
     av_assert0(lltask);
     task = lltask->task;
+    if (ov_model->va_context) {
+        status = fill_vaapi_blob(ov_model, task, request);
```

Comment:
It looks `fill_vaapi_blob` returns int, while you need IEStatusCode? Please align them.
And I would not recommand you use IEStatusCode as inner function return type.

Reply:
Got it, I will use `DNN_GENERIC_ERROR` as return in function `fill_vaapi_blob` when error encounted. And I modifiy the code like this:
```suggestion
        if (fill_vaapi_blob(ov_model, task, request) == DNN_GENERIC_ERROR) {
            av_log(ctx, AV_LOG_ERROR, "Failed to fill va surface blob\n");
            return DNN_GENERIC_ERROR;
        }
```